src/ast/types/blocks-and-statements.ts

```diff
@@ -142,13 +142,13 @@ export interface ClassInstanceCreationExpression extends BaseNode {
 export interface Literal extends BaseNode {
   kind: "Literal";
   literalType:
-    | IntegerLiteral
-    | FloatingPointLiteral
-    | BooleanLiteral
-    | CharacterLiteral
-    | TextBlockLiteral
-    | StringLiteral
-    | NullLiteral;
+  | IntegerLiteral
+  | FloatingPointLiteral
+  | BooleanLiteral
+  | CharacterLiteral
+  | TextBlockLiteral
+  | StringLiteral
+  | NullLiteral;
```

Comment:
Not sure if you want to reformat this, same for the `operator` field in `Assignment`.